src/schema/godkjennEgenPlanAGSchema.ts

```diff
@@ -0,0 +1,9 @@
+import { boolean, object, z } from "zod";
```

Comment:
Vi trenger ikke å opprette schema for typer som vi ikke motar fra backend. Typen kan opprettes direkte i mappen ´src/types`